android-ndk-sys/src/lib.rs

```diff
@@ -36,3 +36,7 @@ include!("ffi_i686.rs");
 
 #[cfg(all(any(target_os = "android", test), target_arch = "x86_64"))]
 include!("ffi_x86_64.rs");
+
+#[cfg(target_os = "android")]
+#[link(name = "android")]
+extern "C" {}
```

Comment:
What about link with _log_ too?

Reply:
log is not required as it's not used in this crate

Reply:
log is not required by this crate, it's added in `android-glue`.